docs/DeveloperGuide.adoc

```diff
@@ -295,51 +297,83 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |student with subjects of different language mediums |add flashcards of differing languages  |I can customise my flashcards to the subject I am interested to do
 
-|`* * *` |user |add a new person |
+|`* * *` |user creating flash cards |save collections of thematically similar flash cards | I can share my cards easily to other users
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user that takes flash card tests |select an option out of four that best matches the front of a flash card |I can check that I have understood the concept the flash card aims to impart
```

Comment:
I think you can make the user story less specific e.g. an option "out of four" doesn't need to be specified, and the purpose can be shortened to simply "I can test my conceptual understanding"